frontend/src/js/controllers/challengeCtrl.js

```diff
@@ -340,7 +341,11 @@
                             vm.displayDockerSubmissionInstructions(vm.page.is_docker_based, vm.isParticipated);
                             utilities.hideLoader();
                         },
-                        onError: function() {
+                        onError: function(response) {
+                            var error = response.data;                            
+                            $rootScope.notify("error", error.detail);
+                            vm.emailVerification = !true;
```

Comment:
@Suryansh5545 Can we please directly set it to false here?

Reply:
done